chart/templates/deployment.yaml

```diff
@@ -22,7 +22,6 @@ spec:
       {{- end }}
       labels:
         {{- include "chart.labels" . | nindent 8 }}
-        app.kubernetes.io/name: federation-controller
```

Comment:
Is it intentional removal? I would keep standard k8s labels.

Reply:
I removed it from this template, because it's already included in `chart.selectorLabels`, which are included in `chart.labels`.